src/Port.js

```diff
@@ -0,0 +1,16 @@
+class Port {
+    constructor (name) {
+        this.name = name;
+        this.ships = [];
+    }
+    addShip(ship) {
+        this.ships.push(ship);
+    }
+    removeShip(ship) {
+        let index = this.ships.indexOf(ship)
+        this.ships.splice(index, 1);
+        ;
```

Comment:
This could be replaced by one line, you dont need to create the extra variable since it is only used, and its content is not long enough to make it unreadable putting it inline:
```
        this.ships.splice(this.ships.indexOf(ship), 1);
```